app/react/App/scss/modules/_review.scss

```diff
@@ -247,7 +243,7 @@ $header-height: 44px;
   }
 
   .settings-footer {
-    @extend .sidepanel-footer;
+    // @extend .sidepanel-footer;
```

Comment:
Please, remove comments from line 56, 181 and 246.

Reply:
done